src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1733,6 +1733,37 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Do not activate when Game Mode is on</value>
     <comment>"Game mode" is the Windows feature to prevent notification when playing a game.</comment>
   </data>
+  <data name="MouseUtils_FindMyMouse_BackgroundColor.Header" xml:space="preserve">
+    <value>Background color</value>
+  </data>
+  <data name="MouseUtils_FindMyMouse_SpotlightColor.Header" xml:space="preserve">
+    <value>Spotlight color</value>
+  </data>
+  <data name="MouseUtils_FindMyMouse_OverlayOpacity.Header" xml:space="preserve">
+    <value>Overlay opacity</value>
+  </data>
+  <data name="MouseUtils_FindMyMouse_OverlayOpacity.Description" xml:space="preserve">
+    <value>Opacity percentage for the overlay</value>
```

Comment:
Isn't the title already self-explainatory? Doesn't add a lot and basically says the same thing twice.

Reply:
Both @Jay-o-Way and @htcfreek asked for it:
- https://github.com/microsoft/PowerToys/pull/14590#issuecomment-976192150
- https://github.com/microsoft/PowerToys/pull/14590#issuecomment-976245073

Reply:
@niels9001  , so perhaps if I just add make the description "Percentage"? Does this make sense?

Reply:
What about simply adding the unit to the end of the first text?
`Spotlight radius (px/pixels)`
`Spotlight inital zoom (factor)`
`Overlay opacity (%/percentage)`
`Animation duration (ms/milliseconds)`

\#KISS (keep it short and simple)

Reply:
I think this should be resolved in a way that it's flexible and can be used across all modules (e.g. setting the opacity for Shortcut Guide overlay, or Zone Opacity for Fancy Zones).

One elegant solution could be adding it to the tooltip:
![image](https://user-images.githubusercontent.com/9866362/143048971-e50a8deb-8e60-4ebe-b6ea-e4e27028c2fb.png)

Other option is to add it after the input control. E.g.:
[ NumberBox ] ms
[ Slider ] %

I'll create a tracking issue for this so we can keep this as is and @jaimecbernado can continue doing awesome things :)!

Reply:
Thank you 😄 
So I've removed the description for now.
PTAL and approve if you think it's ready 😉 
![image](https://user-images.githubusercontent.com/26118718/143051725-80550eca-48f1-4edc-83e3-10d5fc6b7131.png)